.scalafmt.conf

```diff
@@ -20,6 +22,7 @@ project.excludeFilters = [
   "core/shared/src/main/scala-2.x/zio/VersionSpecific.scala"
 ]
 
+rewrite.redundantBraces.generalExpressions = false
```

Comment:
In scalafmt 2.7.0 if-else braces will be extracted to [rewrite.redundantBraces.ifElseExpressions = false](https://scalameta.org/scalafmt/docs/configuration.html#redundantbraces), so you can enable it again if you want to remove braces around `while` one-liners for example